stories/atoms/components/typography/Typography.stories.mdx

```diff
@@ -246,6 +246,23 @@ Internally, this parameter will actually select a variant of the selected font.
   </Story>
 </Canvas>
 
+## Text decoration
+
+It is possible to add a text decoration with the optional property `textDecoration`.  
+This property can take the following value :
```

Comment:
Since the underline value is an enum and not a boolean, we expect to have more values proposed.
So, it could be nice to add a sentence explaining that there will be more choices as the library develops.
What do you think?

Reply:
You're right